docs/webhost api.md

```diff
@@ -0,0 +1,72 @@
+# API Guide
+
+Archipelago has a rudementary API that is able to be queried by endpoints.
+The API is a work-in-progress and should be improved over time.
+
+All API requests will be formatted as: `https://<Archipelago URL>/api/<endpoint>`
+
+## Datapackage Endpoints
+These endpoints are used by applications to aquire and validate they have a current datapackage for item and location IDs.
+
+`/datapackage`  
+Fetches the current datapackage from the WebHost.
+
+`/datapackage/<string:checksum>`    
+Fetches databackage by checksum.
+
+`/datapackage_checksum`  
+Fetches checksum of current datapackage.
+
+## Generation Endpoint
+These endpoints are used internaly for the WebHost to generate games, and validate their generation.
```

Comment:
internally* but also all of these endpoints are accessible externally and are already in use in that way.